Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: server
   repository: ""
-  version: 0.2.0
+  version: 0.3.0
```

Comment:
Version bumps like these are done automatically during the release - no need to do it here :)

Reply:
@genofire If you want to bump the subchart, this should be 1.0.0 due to the breaking changes.